src/ChronosInterval.php

```diff
@@ -134,13 +134,13 @@ public function __construct($years = 1, $months = null, $weeks = null, $days = n
      * syntax as it allows you to do ChronosInterval::create(1)->fn() rather than
      * (new ChronosInterval(1))->fn().
      *
-     * @param int $years The year to use.
-     * @param int $months The month to use.
-     * @param int $weeks The week to use.
-     * @param int $days The day to use.
-     * @param int $hours The hours to use.
-     * @param int $minutes The minutes to use.
-     * @param int $seconds The seconds to use.
+     * @param int|null $years The year to use.
```

Comment:
Yes, I know. But you can pass (and you do) null as well, see the rest of the call code. So it has to be consistent here, as well.